genie-core/src/main/java/com/netflix/genie/core/properties/JobsUsersActiveLimitProperties.java

```diff
@@ -34,7 +35,23 @@
 @Getter
 @Setter
 @Validated
-public class JobsUsersActiveLimitProperties {
+public class JobsUsersActiveLimitProperties implements EnvironmentAware {
+
+    /**
+     * The property prefix for job user limiting.
+     */
+    public static final String PROPERTY_PREFIX = "genie.jobs.users.active-limit";
```

Comment:
In 3.3.x wasn't this `activeLimit`? Will this necessitate changes for everyone? Not sure if spring Boot 1.5 will treat it the same as 2.0 does.

Reply:
This is wrong indeed. I cherry-picked and missed the fact this does something different..